nexus/src/app/background/tasks/support_bundle_collector.rs

```diff
@@ -290,7 +290,7 @@ impl SupportBundleCollector {
             .support_bundle_list_assigned_to_nexus(
                 opctx,
                 &pagparams,
-                self.nexus_id,
+                None,
```

Comment:
Old: Each nexus queries their **own** set of bundles to see what should be destroyed.
New: **All** nexuses query any bundles in these states, and try to clean them up concurrently.

This **does** mean bundle deletion may happen concurrently, from multiple distinct Nexuses, since there is no real meaning of "ownership" after collection has completed.

I think this **should** be safe - the process of deleting a bundle involves:

1. Delete from the sled where it's stored
2. Delete from the database, updating the record

Both of which should be independently safe from concurrent Nexuses